core/src/main/java/hudson/model/Actionable.java

```diff
@@ -74,6 +74,14 @@
     @Deprecated
     @Nonnull
     public List<Action> getActions() {
+       return getPersistedActions();
+    }
+
+    /**
+     * Historically child classes override and return {@link hudson.model.Actionable#getActions()} as
+     * unmodifiable list, while addAction, replaceAction depends on them.
```

Comment:
```suggestion
     * unmodifiable list, while {@link #addAction}, {@link #replaceAction}, etc. depend on them.
```

Reply:
i removed this description at all in new commit. New method is pretty clear.